enterprise/releases/2025-03-0.mdx

```diff
@@ -0,0 +1,84 @@
+---
+title: '2025.03.0 Release'
+backlink_href: /docs/enterprise/releases
+backlink_title: 'pganalyze Enterprise Server: Release Changelog'
+---
+
+**Docker tag:** `quay.io/pganalyze/enterprise:v2025.03.0`
+
+## Notes
+
+- This release requires a database migration when coming from older releases, be sure to follow the [recommended upgrade steps](/docs/enterprise/upgrade)
+- **Important:** If you are currently running v2024.04.X or older, we highly recommend first upgrading to v2024.10.0 and staying there for 7 days
+  - The v2024.10.0 release introduced a backend change for how Log Insights data is stored. Starting that release the old format was still supported in a dual write mode.
+  - Starting with this release the new format is required and the only one supported.
+  - If you have not been on v2024.10.0 for sufficient time the migrations will fail with an error to prevent accidental loss of Log Insights data.
+- **Important:** If you are currently running v2023.08.0 or older, we highly recommend first upgrading to [v2024.04.5](/docs/enterprise/releases/2024-04-5) and staying there for 7 to 35 days
```

Comment:
any reason that you added the link here?

Reply:
I felt it was helpful to reference back the specific release we're asking you to run, especially since its not the most recent one (for the query stats format changes).